### DISCUSSION TITLE
How to use Inquiries Switch

### DISCUSSION THREAD

**Discussion by jcastro** [mKeRix/room-assistant#469] [Q&A]
Hello guys,

I'm currently using monitor to get home-level presence detection and I have started investigating room-assistant for room presence level. So far I got it working and configured it successfully, however I'm having a hard time understanding the Inquiries Switch.

Currently with monitor I trigger the arrival or depart scan when a sensor is activated, so it's not all the time scanning. I would like to do the same with room-assistant however I'm not sure if there's a topic I can trigger to activate the room-assistant scan? I couldn't find or understand it from the help, sorry not a developer :(

My current configuration looks like this

`global:
  instanceName: oficina
  integrations:
    - homeAssistant
    - bluetoothClassic
homeAssistant:
  mqttUrl: 'mqtt://redacted:1883'
  mqttOptions:
    username: redacted
    password: redacted
bluetoothClassic:
  minRssi: -10
  hciDeviceId: 0
  preserveState: true
  inquireFromStart: true
  addresses:
    - XX:XX:XX:XX:XX:B0
    - XX:XX:XX:XX:XX:B0`

Thanks a lot!

**Comment by mKeRix** [comment]
The way you would achieve this in room-assistant is by setting `inquireFromStart` to `false` in the bluetoothClassic integration settings and then writing an automation that calls `switch.turn_on` and, after some time, `switch.turn_off` for all the inquiries switches. You can trigger that with the sensors as before. Another pro-tip: you can set `preserveState` to `true` so that the entities stay in their last state when the inquiries switch is turned off, otherwise they will go to `not_home`.

**Reply by jcastro** [reply]
thank you! I get how it works now!
